From: George Hopkins <george-hopkins@null.net>
Date: Sun, 28 Dec 2014 21:28:39 +0100
Subject: Adjust setup.py scripts

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 9078008..6eacf2a 100644
--- a/setup.py
+++ b/setup.py
@@ -28,6 +28,6 @@ setup(
         'pyserial',
     ],
     scripts=[
-        'esptool.py',
+        'esptool',
     ],
 )
